### DISCUSSION TITLE
Avro messages with TopicRecordNameStrategy

### DISCUSSION THREAD

**Discussion by rackom** [provectus/kafka-ui#3349] [Q&A] [answered]
Hello, I am unable to configure `kafka-ui` with Schema Registry that uses `TopicRecordNameStrategy` for values. In sidebar I can see `Schema Registry` option with all my schemas, but in actual topic messages there is no option to select data type AVRO. All the rest of options are there but this one is missing. I tried to play around with `KAFKA_CLUSTERS_0_SCHEMANAMETEMPLATE` but without any luck. Any suggestions, or I am trying to do something that is not implemented yet?

**Comment by iliax** [comment, marked as answer]
Hello @rackom, 
for deserialization TopicRecordNameStrategy should work fine on `master` image. (we have an issue with it in `latest` that is fixed updated yet).
Can you please check master image?

**Reply by rackom** [reply]
Thank you very much, indeed I was using `latest`, works fine with `master`. :)

**Reply by supportietf** [reply]
Yes
